sql/updates/world/2020_02_28_00_world.sql

```diff
@@ -0,0 +1,82 @@
+UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='50' WHERE `entry`=2957 AND `item`=33009;
+UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='50' WHERE `entry`=3068 AND `item`=33009;
+UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='50' WHERE `entry`=2956 AND `item`=33009;
```

Comment:
with this code, meat will drop even if quest not active, is that legit ? quest items should use negative questchance to drop only if quest is active, and increasing the droprate from 10% to 50% is 5times more often.

Reply:
in other words change it from -10 to 10 if the drop is not required to have quest active, to represent a kill 10npc quest.

or if the drop requires quest status active then use questchance -50 instead of chance.

Reply:
> with this code, meat will drop even if quest not active, is that legit ? people who didnt complete/did complete the quest will get spammed by this quest item, quest items should use negative questchance if the item is reqitem in quest_template to drop only if quest is active, and increasing the droprate from 10% to 50% is 5times more often, lets say we kill 50 wolfs 10% drop is 5 meat = 1 meat per 10 npc sounds to me like a normal kill 10 mobs quest, if you increase the droprate to 50% its 25 meat per 50 wolf means its like 2.5 wolfs per meat that does not sound like a kill quest.

you just need 1 meat not more, in the videos on YouTube, they all get meat after killing first mob. However, after reviewing lower patches, I found that 50% is best option.
And yes i must change to -50 , Sorry for this mistake

Reply:
no -50 is wrong because the quest objective is not reqitem in this particular quest

Reply:
because the quest objective is not req item you need to set it as positive and add loot condition for the quest.

Reply:
> 
> 
> no -50 is wrong because the quest objective is not reqitem in this particular quest

it's required item

Reply:
no its not reqitem the quest objective is killcredit

Reply:
> 
> 
> because the quest objective is not req item you need to set it as positive and add loot condition for the quest.

You need to find this item first and you need to have Quest for find this item.
If you do not have item you cannot complete Quest.

Reply:
the quest objective is kill credit, you loot the item and use it, it gets consumed on use. and is not required for quest completion.

Reply:
> 
> 
> the quest objective is kill credit, you loot the item and use it, it gets consumed on use. and is not required for quest completion.

If you don't have item, how do you want to complete it?

Reply:
the system iterate the required item field in quest_template to check if the item is in quest_template or is questobjective_item to assign the item to a quest in this particular quest the item is not in the template and is not questobjective_item so the loot cant assign a quest to the item thats why it never will drop if its negative. and we need to set it positive and manualy add a quest condition.

Reply:
> 
> 
> the system iterate the required item field in quest_template to check if the item is in quest_template or is questobjective_item to assign the item to a quest in this particular quest the item is not in the template and is not questobjective_item so the loot cant assign a quest to the item thats why it never will drop if its negative. and we need to set it positive and manualy add a quest condition.

but added item id in RequiredSourceItem field, must drop in npc :| i check again.

Reply:
> 
> 
> the system iterate the required item field in quest_template to check if the item is in quest_template or is questobjective_item to assign the item to a quest in this particular quest the item is not in the template and is not questobjective_item so the loot cant assign a quest to the item thats why it never will drop if its negative. and we need to set it positive and manualy add a quest condition.

According to HasQuestForItem function in player.cpp of line 17446 onwards, if RequiredSourceItemId is added in database field, it must be show item for player during loot.
```
for (uint8 j = 0; j < QUEST_SOURCE_ITEM_IDS_COUNT; ++j)
{
    // examined item is a source item
    if (qInfo->RequiredSourceItemId[j] == itemId)
    {
        ItemTemplate const* pProto = sObjectMgr->GetItemTemplate(itemId);

        // 'unique' item
        if (pProto->MaxCount && int32(GetItemCount(itemId, true)) < pProto->MaxCount)
            return true;

        // allows custom amount drop when not 0
        if (qInfo->RequiredSourceItemCount[j])
        {
            if (GetItemCount(itemId, true) < qInfo->RequiredSourceItemCount[j])
                return true;
        }
        else if (GetItemCount(itemId, true) < pProto->GetMaxStackSize())
            return true;
    }
}
```

my say is true?

Reply:
meat = unique item so,
 max count = 1, GetItemCount = 1.

if (getitemcount < maxcount)
    return true;

Reply:
> 
> 
> meat = unique item so,
> max count = 1, GetItemCount = 1.
> 
> if (getitemcount < maxcount)
> return true;

So it has to be added in loot, I don't know why it's not added.

Reply:
issue is fixed